static/app/views/issueDetails/groupTagValues.tsx

```diff
@@ -281,7 +293,7 @@ class GroupTagValues extends AsyncComponent<
   }
 }
 
-export default withSentryRouter(GroupTagValues);
+export default withSentryRouter(withOrganization(GroupTagValues));
```

Comment:
can you `useOrganization` instead of `withOrganization`?

Reply:
wait but this is a class component?

Reply:
Oops you're right lol